File: C++/1038.cpp
#include <stdio.h>

int main() {
 int cod,qtd;
 
 scanf("%d %d",&cod,&qtd);
 if (cod==1) printf("Total: R$ %.2f\n", qtd*4.0);

 if (cod==2) printf("Total: R$ %.2f\n", qtd*4.5);
 
 if (cod==3) printf("Total: R$ %.2f\n", qtd*5.0);
 
 if (cod==4) printf("Total: R$ %.2f\n", qtd*2.0);
 
 if (cod==5) printf("Total: R$ %.2f\n", qtd*1.5);
 
 
 return 0;
 		
   
}

File: Python/1038.py
a, b = input().split()
a = int(a)
b = int(b)

if a==1 :
    c = b*4.0
elif a==2 :
    c = b*4.5
elif a==3 :
    c = b*5.0
elif a== 4 :
    c = b*2.0
elif a==5 :
    c = b*1.5

print(f'Total: R$ {c:.2f}')